providers/providers.go

```diff
@@ -24,6 +24,8 @@ type Provider interface {
 // New provides a new Provider based on the configured provider string
 func New(provider string, p *ProviderData) Provider {
 	switch provider {
+	case "liquid":
+		return NewLiquidProvider(p)
```

Comment:
Please comment out all the other providers from this `switch`. You can leave the code in, same idea as above, it simpler to rebase our changes over updates if the code files are still there.